Simplenote/NoteContentHelper.swift

```diff
@@ -0,0 +1,56 @@
+import Foundation
+
+// MARK: - NoteContentHelper
+//
+struct NoteContentHelper {
+
+    private init() {
+
+    }
```

Comment:
Trick! (standard across the division!):

```
enum NoteContentHelper { }
```

That way it'd be impossible to initialize a NoteContentHelper (unless... you forget not to add a `case`!).

That'd be why there are so many `enum Metrics` / `enum Constants` / `enum Settings` here and there!

Reply:
(Plus) please relocate this file to `Simplenote/Classes`

Reply:
Good point, thanks!